lib/build/cpp/c++

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
```

Comment:
I'm not too tied to having this as a separate file, I mostly did it this way for now just to keep things separate whilst we're still figuring bits out. It does possibly seem slightly nicer (and kind of inline with usual linux conventions from what I can tell), to have a separate `c++` wrapper script.